devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -107,6 +107,20 @@ def parse(String description) {
 	Map eventMap = zigbee.getEvent(description)
 	Map eventDescMap = zigbee.parseDescriptionAsMap(description)
 
+	if ( device.getDataValue("deviceJoinName") == "eZEX Switch 1") {
```

Comment:
@inasail can you verify that this value does not change if you rename the device?

normally we would do this using a manufacturer check, but eZex did not include that string for whatever reason...

Reply:
@greens, I tested with other sensors like moisture, outlet sensor, but in these both cases, the deviceJoinName is returned with 'null' string(even it is defined in the fingerprint), so I can't trust this value in this DTH. I used this condition not to effect other sensors in this DTH. but I think it is okay to adjust as common logic. So I will delete this checking condition.